xypath/loader.py

```diff
@@ -0,0 +1,31 @@
+import messytables
+import xypath
+
+def table_set(filename):
```

Comment:
I'm a bit confused by the interface here, in particular the relationship between `table_set` and `get_sheets`.

I'd expect table_set to return all the tables (table == sheet in my mental model!), I don't understand why there's a second function that can then return multiple things.